### ISSUE TITLE
Test wether KNITRO can be used

### ISSUE THREAD

**Issue by tmigot** [jump-dev/KNITRO.jl#205]
Depending on the user's configuration, KNITRO might be available or not. What is the preferable way to check this?

Right now, I am using a try/catch structure, but I was wondering if there was a cleaner solution.
```
try
  using KNITRO
catch
  # use something else
end
```

**Comment by odow** [comment]
See https://github.com/JuliaPackaging/Requires.jl

**Comment by frapac** [comment]
We could adopt a similar solution as in CUDA.jl, and implement a function `has_knitro()` such that
```julia
KNITRO.has_knitro()
```
returns `true` if KNITRO is available. Would that solve your issue?

**Comment by frapac** [comment]
Closed by #219
